ebsco_adapter/ebsco_adapter/test.Dockerfile

```
@@ -0,0 +1,9 @@
+FROM python:3.8
```

We are liable to get bitten by Docker Hub rate limits, elsewhere we use the ECR Public Gallery mirrors of docker hub official images
```suggestion
FROM public.ecr.aws/docker/library/python:3.8-slim
```

---

Also - can we use a more recent version (for the Lambda as well)?